content/threads_intro.tex

```diff
@@ -0,0 +1,73 @@
+This section specifies the interaction between \openshmem{} interfaces and the
+user threads, and also describes the routines that can be used for initializing and 
+querying the thread environment.
+ 
+
+\begin{itemize}
+
+\item
+In a thread-compliant implementation, an \openshmem{} \ac{PE} is multithreaded. 
+Each thread can issue \openshmem{} calls; however the threads
```

Comment:
That still seems to conflict with, say, the FUNNELED model. A program using the FUNNELED model is an example of a multi-threaded OpenSHMEM PE, but _only one (the main) thread_ can issue OpenSHMEM calls.

What is your intent in establishing the term "thread-compliant implementation"? Assuming that an updated version of this PR is accepted for 1.4, aren't _all_ implementations required to provide, for example, `shmem_init_thread()`?

I think a "thread-_supporting_ implementation" might be better defined as one that supports a threading model higher (need a better word here) than SHMEM_THREAD_SINGLE. That said, as a user, I don't want any implementation to take the low-bar out and say "we support threads!" but only provide FUNNELED. I'm not sure establishing such a term is necessary here.